changelog

```diff
@@ -156,50 +156,50 @@ Version 0.9.15:
       * The intro mission now mentions the usefulness of buying local maps. (@Hecter94)
-      * Improved the salaries that you get during the Free Worlds campaign to better cover the ships that you're expected to have at the time. (@Anarchist2)
+      * Improved the salaries you get during the Free Worlds campaign to better cover the ships you're expected to have at the time. (@Anarchist2)
       * The final Kestrel mission now displays the thumbnail of the Kestrel you chose. (@Anarchist2)
       * Increased how much the Coalition pays for Yottrite from 220k per sample to 350k per sample. (@Arachi-Lover)
-      * Updated the prisoner transports in some Free Worlds missions to actually be equipped with brigs. (@AlbertNewton)
+      * Updated the prisoner transports in some Free Worlds missions to be equipped with brigs. (@AlbertNewton)
       * The Remnant missions that ask you to get a copy of Alexandria's archives now make it clear that you keep a copy for yourself. (@Amazinite)
       * The "Deep: Remnant" missions are no longer required to trigger "Deep: Scientist Rescue", which is the set of missions that unlocks the Bactrian. (@Amazinite)
       * Improved the dialog in one of the new Remnant missions to make it more understandable. (@alextd)
       * Rewrote part of "FW Hope Recon 1C" to have the Gunboat join the Oathkeepers should you ask them to surrender. (@jostephd)
       * The senator in "FW Senate 1B" now commends you if you reach the destination in an impossibly short time. (@Hecter94)
-      * The corporate espionasge job now has a short deadline, as it is very easy to miss the objective then never find the npc that you're suppose to scan. (@Terin)
+      * The corporate espionage job now has a short deadline, as it is easy to miss the objective and never find the NPC that you're supposed to scan. (@Terin)
       * Slight wording change in FW Bounty 1 to reduce repeated information. (@Anarchist2)
     * Balance:
       * Altered the default equation for when a ship becomes disabled to become continuous. The lowest-hull ships still become disabled at about 45% remaining hull, but the highest-hull ships approach a threshold of 10% hull remaining instead of all ships over 4,500 hull becoming disabled at 15%. This results in most human ships becoming disabled slightly earlier while higher health alien ships become disabled slightly later. (@Ferociousfiend)
-      * Buffed the Heavy Shuttle while also increasing its price to make it more of a midpoint between the Shuttle and the Scout or Bounder rather than having a massive price gap between the Shuttle and Heavy Shuttle and the aforementioned ships. (@Zitchas)
+      * Buffed the Heavy Shuttle while increasing its price to make it more of a midpoint between the Shuttle and the Scout or Bounder rather than having a massive price gap between the Shuttle and Heavy Shuttle and the ships mentioned above. (@Zitchas)
```

Comment:
```suggestion
      * Buffed the Heavy Shuttle while increasing its price to make it more of a midpoint between the Shuttle and the Scout or Bounder rather than having a massive price gap between the Shuttle and Heavy Shuttle and the ships mentioned previously. (@Zitchas)
```